src/services/octokit.ts

```
@@ -21,17 +24,20 @@ export async function fetchUserGitHubNotifications(
+    const { data: githubNotifications } =
+      await octokit.activity.listNotificationsForAuthenticatedUser({
+        all: true,
+        participating: false,
+        since: new Date(
+          Date.now() - daysBack * 24 * 60 * 60 * 1000
+        ).toISOString(),
+        before: new Date().toISOString(),
+      });
+    return { data: githubNotifications };
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)